libse/Utilities.cs

```diff
@@ -901,14 +901,13 @@ public static bool IsValidRegex(string testPattern)
 
             try
             {
-                Regex.Match(string.Empty, testPattern);
+                return Regex.IsMatch(string.Empty, testPattern);

```

Comment:
> Are you sure this is correct?

yeah, Both I'll throw:

**ArgumentException** :  A regular expression parsing error occurred. :)
**ArgumentNullException**: input or pattern is null.
**RegexMatchTimeoutException**: A time-out occurred. For more information about time-outs, see the Remarks section.

* [Regex.Match](https://msdn.microsoft.com/en-us/library/0z2heewz%28v=vs.110%29.aspx)
* [Regex.IsMatch](https://msdn.microsoft.com/en-us/library/sdx2bds0%28v=vs.110%29.aspx)

Also stated in MSDN that:
**The IsMatch method is typically used to validate a string or to ensure that a string conforms to a particular pattern without retrieving that string for subsequent manipulation.**